Detectors/ZDC/workflow/src/ZDCDataReaderDPLSpec.cxx

```diff
@@ -78,16 +78,56 @@ void ZDCDataReaderDPLSpec::run(ProcessingContext& pc)
     mRawReader.setVerifyTrigger(mVerifyTrigger);
     LOG(info) << "Check of trigger condition during conversion is " << (mVerifyTrigger ? "ON" : "OFF");
   }
+
   uint64_t count = 0;
+  uint64_t nErr[3] = {0};
```

Comment:
this should be `static uint64_t nErr[3] = {0};`

Reply:
Should it survive different run calls to drop all the remaining timeframe?